### ISSUE TITLE
Add Windows compatibility

### ISSUE THREAD

**Issue by retrography** [retrography/pRDS#8]
Alternatively check for the OS and fall back to R implementation on Windows

1. Turns out Rtools installs a full Sys2 / Mingw setup, including the file utility: C:\Program Files\rtools40\usr\bin\file
2. `pkgbuild::rtools_path()` gives us the path to rtools usr/bin directory
3. zstd works for parallel gzip operations on windows out of the box (chocolatey)
4. 7z works for parallel xz operations on windows out of the box (chocolatey)
5. 7z works for parallel bzip2 operations as well

**Comment by retrography** [comment]
Adding rtools to path

```R
path <- Sys.getenv("PATH")
rt <- gsub('/','\\\\',pkgbuild::rtools_path())
path <- paste(rt, path, sep=";")
```

**Comment by retrography** [comment]
Addressed in ec25d2ed2fc007e975e58820f16dce56cd04fbe5
